File: Competitive Programming 3/Section 1.3.3/11364 - Parking/Python/main.py
def main():
    t = int(input())
    for _ in range(0,t):
        _ = int(input())
        numbers = map(int,input().split())
        minimum,maximum = 99,0
        for number in numbers:
            minimum = min(number,minimum)
            maximum = max(number,maximum)
        print(maximum - minimum + maximum - minimum)
main()

File: Competitive Programming 3/Section 1.3.3/11364 - Parking/C++/main.cpp
#include <bits/stdc++.h>

using namespace std;

int t,n,x;

int main(){
    ios_base::sync_with_stdio(0);cin.tie(0);
    cin >> t;
    while(t--){
        cin >> n;
        int minimum = 99 , maximum = 0 ;
        for(int i = 1; i <= n ; i ++){
            cin >> x ; 
            minimum = min(minimum,x), maximum = max(maximum,x);
        }
        cout << maximum - minimum + maximum - minimum << "\n";
    }
    return 0;
}